test/ct/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM artifactory.algol60.net/csm-docker/unstable/hms-test:3.0.0-20220225025438.c37b99e
```

Comment:
Need to change this!

Reply:
@rsjostrand-hpe is there a way to make this link not look like : 

```
FROM artifactory.algol60.net/csm-docker/stable/hms-test:3.0.0-20220228165848.1738330
```

and more like `FROM artifactory.algol60.net/csm-docker/stable/hms-test:3.0.0`?

Reply:
Yes, you need to make a stable release in the hms-test repo.